www/activities/settings/views/statistics.html

```diff
@@ -72,6 +72,40 @@
             </div>
           </div>
         </li>
+
+        <li>
+          <div class="item-content">
+            <div class="item-inner">
+              <div class="item-title" data-localize="settings.statistics.send-data">Send data to external service</div>
```

Comment:
You should add the locale strings to the `locale-en.json` file ([here](https://github.com/Kroseida/waistline/blob/6df9f359273325f6e239a0a6e7df0241d95f7e69/www/assets/locales/locale-en.json#L293)) so they can be translated by other users.

Reply:
done